specification/cognitiveservices/ContentSafety/routes.tsp

```diff
@@ -148,3 +165,22 @@ interface TextBlocklists {
     ListQueryParametersTrait<StandardListQueryParameters>
   >;
 }
+
+@added(Versions.v2024_02_15_Preview)
+interface TextGroundednessDetectionOperations {
+  @summary("Detect Groundedness")
+  @doc("A synchronous API for the analysis of language model outputs to determine alignment with user-provided information or identify fictional content.")
+  @route("/text:detectGroundedness")
+  @post
+  detectGroundednessOptions is Azure.Core.RpcOperation<
```

Comment:
It seems you did not add this operation to client.tsp?

If you need SDK for this preview, you need to add it there.